lib/rules/key-spacing.js

```diff
@@ -333,11 +336,18 @@ module.exports.schema = [
             "align": {
                 "enum": ["colon", "value"]
             },
+            "mode": {
+                "enum": ["strict", "minimum"]
+            },
             "beforeColon": {
-                "type": "boolean"
+                "type": ["boolean", "number"],
```

Comment:
Why do we need this to be a number as well? From the issues it sounded like `strict/minimum` would cover the use cases.